src/renderer/components/number-results.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+export interface NumberResultsProps {
+	numberOfResults: number | undefined;
```

Comment:
issue: why is the prop `numberOfResults` potentially undefined if `LogContentState.numberOfResults` has type `number`?

Reply:
Forgot to remove that, before it would give an error when I didn't add undefined to it but I resolved it

Reply:
Fixed